bash_completions.md

```diff
@@ -301,6 +303,8 @@ So while there are many other files in the CWD it only shows me subdirs and thos
 
 As for nouns, Cobra provides two ways of defining dynamic completion of flags.  Note that both these methods can be used along-side each other as long as they are not both used for the same flag.
 
+**Note**: *Custom Completions written in Go* will automatically work for other shell-completion scripts (e.g., Fish shell), while *Custom Completions written in Bash* will only work for Bash shell-completion.  It is therefore recommended to use *Custom Completions written in Go*.
```

Comment:
These seems to be duplicated above

Reply:
I thought that if someone went straight the the completion of flags section, without reading the entire page, it would useful go guide them towards the Custom Go completion and not the bash-specific completion.

If you find that unnecessary, I will remove it.  Just let me know.

Reply:
Ah I see.  We can keep it.